osu.Game/Screens/Menu/OsuLogo.cs

```diff
@@ -35,6 +35,12 @@ public partial class OsuLogo : BeatSyncedContainer
 
         private const double transition_length = 300;
 
+        /// <summary>
+        /// The osu! logo sprite has a shadow included in its texture.
+        /// This adjustment vector is used to match the precise edge of the border of the logo.
+        /// </summary>
+        private static readonly Vector2 scale_adjust = new Vector2(0.96f);
```

Comment:
There's two more cases related to the logo that use `0.96f`:
https://github.com/ppy/osu/blob/6b017ac05f0af313d55d482033b60e8f72dcc877/osu.Game/Screens/Menu/OsuLogo.cs#L409

https://github.com/ppy/osu/blob/6b017ac05f0af313d55d482033b60e8f72dcc877/osu.Game/Screens/Menu/IntroWelcome.cs#L134

Reply:
Nice find.